tests/test_main.py

```diff
@@ -42,12 +42,17 @@
 @pytest.fixture
 def base_config_present(create_config, monkeypatch):
     tmp_path = create_config(
```

Comment:
Like in the other files, please ensure you keep versions of these tests that test the same charm directory.

Reply:
OK, I will need to test this.